pkg/provider/azure_loadbalancer.go

```diff
@@ -2198,15 +2206,15 @@ func (az *Cloud) buildHealthProbeRulesForPort(serviceManifest *v1.Service, port
 			}
 		}
 		if path == nil {
-			path = pointer.String(consts.HealthProbeDefaultRequestPath)
+			path = pointer.String("/readyz")
```

Comment:
The kube-proxy health check needs `/healthz` so I think you can revert this line

Reply:
Oops good catch.
Need to change it to `/healthz` then because the default there is `/`